Master does not pass tls tests.

Running `make test` on a clean unmodified master does not pass the tests. Is this functionality that hasn't been developed yet, or a regression?

```
# Starting Test: tls-server-verify - Do not request certs. Everyone is unauthorized.
    connecting with agent1
'openssls_client -tls1 -connect 127.0.0.1:32000 -key /home/aiverson/projects/upstream-luvit/tests/fixtures/keys/agent1-key.pem -cert /home/aiverson/projects/upstream-luvit/tests/fixtures/keys/agent1-cert.pem'
Uncaught Error: ...projects/upstream-luvit/tests/test-tls-server-verify.lua:111: agent1 rejected, but should NOT have been
stack traceback:
        [C]: in function 'assert'
        ...projects/upstream-luvit/tests/test-tls-server-verify.lua:111: in function 'handler'
        /home/aiverson/projects/upstream-luvit/deps/core.lua:248: in function 'emit'
        ...e/aiverson/projects/upstream-luvit/deps/childprocess.lua:128: in function <...e/aiverson/projects/upstream-luvit/deps/childprocess.lua:125>
        [C]: in function 'run'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:79: in function </home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:64>
        [C]: in function 'xpcall'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:64: in function 'run'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:165: in function 'tap'
        /home/aiverson/projects/upstream-luvit/tests/run.lua:42: in function 'fn'
        [string "bundle:deps/require.lua"]:310: in function 'require'
        /home/aiverson/projects/upstream-luvit/main.lua:118: in function 'main'
        /home/aiverson/projects/upstream-luvit/init.lua:49: in function </home/aiverson/projects/upstream-luvit/init.lua:47>
        [C]: in function 'xpcall'
        /home/aiverson/projects/upstream-luvit/init.lua:47: in function 'fn'
        [string "bundle:deps/require.lua"]:310: in function <[string "bundle:deps/require.lua"]:266>
make: *** [Makefile:12: test] Error 255
```

--- a/tests/test-tls-server-verify.lua
+++ b/tests/test-tls-server-verify.lua
@@ -49,7 +49,7 @@ local function optionsIterator(options)
 end
 
 local function runClient(options, port, callback)
-  local args = { 's_client', '-tls1', '-connect', '127.0.0.1:' .. port }
+  local args = { 's_client', '-connect', '127.0.0.1:' .. port }
   print('  connecting with ' .. options.name)
 
   if options.name == 'agent1' then

